packages/tools/pie-icons-webc/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@justeattakeaway/pie-icons-webc",
+  "version": "0.1.0",
```

Comment:
You'll need to set this to be `0.0.0` for the first release, as changesets will then bump it on merge to `v0.1.0`